templates/openstack/pricing-calculator.html

```diff
@@ -129,6 +128,34 @@ <h3 class="p-heading--4">Still not sure which cloud platform suits you best?</h3
 };
 updateFormFields();
 
+const addAnotherInstance = () => {
+  document.addEventListener("contactModalLoaded", (event) => {
+    const button = document.querySelector("#add-another-instance");
+    button.addEventListener("click", () => {
+      const instanceTypes = Array.from(
+        document.querySelectorAll("#instance-type")
+      );
+      const length = instanceTypes.length;
+      for (let i = 0; i < length; i++) {
+        if (instanceTypes[i].classList.contains("u-hide")) {
+          instanceTypes[i].classList.remove("u-hide");
+          //when new instance is added add p-heading--five class to h3s for inputs only to capture values in comment.
+          const inputTitles = instanceTypes[i].querySelectorAll("h3")
+          inputTitles.forEach(inputTitle => {
+            if(!inputTitle.classList.contains("p-heading--four")) {
+              inputTitle.classList.add("p-heading--five")
```

Comment:
If you add a class that exists it does not duplicate the class. So you don't need the conditional above.